app/marketing/management/commands/expiration_start_work.py

```diff
@@ -60,26 +60,49 @@ def handle(self, *args, **options):
         for day in days:
             start_date = (timezone.now() - timezone.timedelta(days=(day+1)))
             end_date = (timezone.now() - timezone.timedelta(days=day))
-            interests_by_created_on = Interest.objects.select_related('profile').filter(
-                created__gte=start_date,
-                created__lt=end_date,
-                pending=False,
-            )
-            interests_by_accepted_on = Interest.objects.select_related('profile').filter(
+
+            profiles = Interest.objects.select_related('profile')
+
+            interests_by_accepted_on = profiles.filter(
                 acceptance_date__gte=start_date,
                 acceptance_date__lt=end_date,
                 pending=False,
             )
-            interests = (interests_by_accepted_on | interests_by_created_on).distinct('pk')
+
+            interests_by_created_on = profiles.filter(
+                created__gte=start_date,
+                created__lt=end_date,
+                pending=False
+            )
+
+            interests = (interests_by_accepted_on | interests_by_created_on)
+            interests = interests.distinct('pk')
             print(f'day {day} got {interests.count()} interests')
             for interest in interests:
-                interest_day_0 = interest.created if not interest.acceptance_date else interest.acceptance_date
-                bounties = Bounty.objects.current().filter(
-                    interested=interest,
-                    project_type='traditional',
-                    network='mainnet',
-                    idx_status__in=['open', 'started']
-                )
+
+                if interest.acceptance_date:
+                    interest_day_0 = interest.acceptance_date
+                    bounties = Bounty.objects.current().filter(
+                        interested=interest,
+                        project_type='traditional',
+                        network='mainnet',
+                        idx_status__in=['open', 'started'],
+                        permission_type='approval'
+                    )
+                else:
+                    interest_day_0 = interest.created
+                    bounties = Bounty.objects.current().filter(
+                        interested=interest,
+                        project_type='traditional',
+                        network='mainnet',
+                        idx_status__in=['open', 'started'],
+                        permission_type='permissionless'
```

Comment:
You should be able to eliminate some of this by:

```python
interest_day_0 = interest.acceptance_date or interest.created  # Will default to acceptance_date if it's not None or fallback to created

bounties = Bounty.objects.current().filter(interested=interest, project_type='traditional', network='mainnet', idx_status__in=['open', 'started'])
```
The `interest` should be unique to the `bounty`, so we shouldn't have to perform a separate query for each type based on `permission_type`

Reply:
Thanks, fixed!